fvm/src/externs/mod.rs

```diff
@@ -23,7 +21,7 @@ pub trait Rand {
     /// ChainEpoch, Entropy from the ticket chain.
     fn get_chain_randomness(
         &self,
-        pers: DomainSeparationTag,
```

Comment:
This is not consensus critical, but it has been on my mind for a while.

1. Removes unnecessary error cases from the kernel.
2. Paves the way for user actors (that won't care about _specific_ tags).

Reply:
> This also means that the type (DomainSeparationTag) can move from fvm_shared to fvm_sdk, or directly to the built-in actors runtime (this option is probably better to avoid confusion).

I've removed the type from the FVM entirely.

Reply:
Nah, not a problem.